.github/workflows/main.yml

```diff
@@ -19,11 +19,15 @@ jobs:
           node-version: '10.18.1'
 
       - name: Install dependencies
-        run: node common/scripts/install-run-rush.js install
+        run: |
+          cd extensions/ql-vscode
+          npm install
         shell: bash
 
       - name: Build
-        run: node common/scripts/install-run-rush.js build
+        run: |
+          cd extensions/ql-vscode
+          gulp
```

Comment:
Can we change this to `npm run build`? This will ensure `$PATH` is correct.

Reply:
Sure, and I agree with similar comments below that confining any call to gulp into the `package.json` scripts is generally a better approach.